src/meta/proto-conv/tests/it/user_proto_conv.rs

```diff
@@ -120,6 +121,7 @@ pub(crate) fn test_s3_stage_info() -> mt::UserStageInfo {
         copy_options: mt::CopyOptions {
             on_error: mt::OnErrorMode::SkipFileNum(666),
             size_limit: 1038,
+            split_size: 0,
```

Comment:
add a test_s3_stage_info_v16(), test_gcs_stage_info_v16() and test_oss_stage_info_v16() for new changes.